src/widgets/oneOf.vue

```diff
@@ -52,7 +68,13 @@ export default {
 	components: {
 		Wrapper,
 	},
-	data: function() {
+	props: {
+		both: {
+			type: Boolean,
+			default: false
+		}
+	},
+	data() {
```

Comment:
the syntax has changed for this

Reply:
Im not sure what you are pointing with this. The data syntax is now as it should be, the older notation should be avoided for consistency.